dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -49,7 +49,7 @@
     dialog.show();
   }
 
-  window.showStartOverDialog = function(callback) {
+  appOptions.dialog.showStartOverDialog = function(callback) {
```

Comment:
Hmm I thought in general we were trying to avoid modifying `appOptions` outside of Rails?  The `dashboard` namespace might be a better place for these (but it's also become sort of a catch-all).